dev/prepare_db.pql

```diff
@@ -4,41 +4,73 @@ func run_sql(code) {
     force_eval( SQL( nulltype, code ))
 }
 
+func drop_table(t) {
+    run_sql("DROP TABLE IF EXISTS " + t)
+}
+
+func create_indices(tbl) {
+    tbl.add_index("id", true)
+    tbl.add_index("timestamp")
+    tbl.add_index(["id", "timestamp"])
+}
+
 // Cleanup
 func cleanup() {
-    run_sql("DROP TABLE IF EXISTS rating")
-    run_sql("DROP TABLE IF EXISTS tmp_rating")
-    run_sql("DROP TABLE IF EXISTS rating_del1")
-    run_sql("DROP TABLE IF EXISTS rating_update1")
-    run_sql("DROP TABLE IF EXISTS rating_update001p")
-    run_sql("DROP TABLE IF EXISTS rating_update1p")
-    run_sql("DROP TABLE IF EXISTS rating_del1p")
-    run_sql("DROP TABLE IF EXISTS rating_update50p")
+    drop_table("rating")
+    drop_table("tmp_rating")
+    drop_table("rating_del1")
+    drop_table("rating_update1")
+    drop_table("rating_update001p")
+    drop_table("rating_update1p")
+    drop_table("rating_del1p")
+    drop_table("rating_update50p")
     commit()
 }
 
 cleanup()
 
 // Import CSV
-if (db_type == "snowflake") {
-    print "Uploading ratings CSV"
-
-    run_sql("RM @~/ratings.csv.gz")
-    run_sql("PUT file://dev/ratings.csv @~")
+if (db_type == "snowflake" or db_type == "redshift") {
```

Comment:
What could we do to remove the Cloud databases as special-cases?

Reply:
I can do something like `db.is_cloud()`, but that would only solve this specific `if`.

I don't see how we can get rid of db-specific code.

What we can do is have an separate module for each db, implementing `import_sample_csv()`, and have the main module call it. So at least the db-specific stuff will be separated.